packages/env/README.md

```diff
@@ -475,7 +475,7 @@ This is useful for plugin development when upstream Core changes need to be test
 
 ```json
 {
-	"core": "WordPress/WordPress#master",
+	"core": "WordPress/WordPress#trunk",
```

Comment:
There is no `trunk`branch in the WordPress repository. It has to stay unchanged.